etc/ci_scripts/dump_artifacts/cli.py

```diff
@@ -27,33 +27,49 @@ def parse_args() -> Any:
     parser = argparse.ArgumentParser(
         description="Dump all Docker logs for a given docker-compose project"
     )
-    parser.add_argument("--compose-project", dest="compose_project", required=True)
+    parser.add_argument(
+        "--compose-project",
+        dest="compose_project",
+        required=False,
+        default=None,
+        help="Docker Compose project. Do not specify if Docker-Compose is not involved (e.g. running against prod)",
+    )
+    parser.add_argument(
+        "--dump-agent-logs",
+        dest="dump_agent_logs",
+        required=False,
+        default=False,
+        help="Dump the logs for Nomad/Consul agents (e.g. running locally)",
+    )
     return parser.parse_args()
 
 
 def main() -> None:
     args = parse_args()
-    compose_project = args.compose_project
+    compose_project: Optional[str] = args.compose_project
+    dump_agent_logs: bool = args.dump_agent_logs
 
     cwd = os.getcwd()
     timestamp = datetime.now().strftime("%Y%m%d%H%M%S")
     artifacts_dir = Path(f"{cwd}/test_artifacts/{compose_project}_{timestamp}")
     os.makedirs(artifacts_dir, exist_ok=False)
 
-    docker_artifacts.dump_all_docker_logs(
-        compose_project=compose_project, artifacts_dir=artifacts_dir
-    )
-    docker_artifacts.dump_volume(
-        compose_project=compose_project,
-        volume_name="dgraph_export",
-        artifacts_dir=artifacts_dir,
-    )
+    if compose_project is not None:
```

Comment:
Maybe just `if compose_project:` ?

Reply:
in this case it's fine. regarding Optional[str] in particular, I've run into cases previously where `""` and `None` should behave differently and over-protect against it